src/test/java/fr/insalyon/creatis/moteur/plugins/workflowsdb/databases/MariaDB.java

```diff
@@ -0,0 +1,42 @@
+package fr.insalyon.creatis.moteur.plugins.workflowsdb.databases;
+
+import java.sql.SQLException;
+
+import org.testcontainers.DockerClientFactory;
+import org.testcontainers.containers.MariaDBContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import fr.insalyon.creatis.moteur.plugins.workflowsdb.Database;
+
+public class MariaDB extends Database {
+
+    public MariaDBContainer<?> mariadb = new MariaDBContainer<>(DockerImageName.parse("mariadb:11.5-ubi"));
+
+    @Override
+    public void create() throws SQLException {
+        mariadb.start();
+        
+        try (var connection = mariadb.createConnection("")) {
+            try (var stmt = connection.createStatement()) {
+                stmt.execute("ALTER DATABASE " + mariadb.getDatabaseName() + " CHARACTER SET latin1;");
+            }
+        }
+        schema = mariadb.getDatabaseName();
+        driverClass = "org.mariadb.jdbc.Driver";
+        url = mariadb.getJdbcUrl();
+        dialect = "org.hibernate.dialect.MySQLDialect";
+        user = mariadb.getUsername();
+        password = mariadb.getPassword();
+        super.create();
```

Comment:
instead of relying on the children classes to call super.create() at the end, i prefer that the parent method call an abstract method like `initProperties`